system-addon/lib/SectionsManager.jsm

```diff
@@ -162,6 +169,11 @@ const SectionsManager = {
               description: card.description,
               previewImageURL: card.image
             });
+            PlacesUtils.history.insert({
+              url,
+              title: card.title,
+              visits: [new Date()]
```

Comment:
Looking at `PlacesUtils.validatePageInfo` looks like it will use now as a default visit date anyway: https://searchfox.org/mozilla-central/source/toolkit/components/places/PlacesUtils.jsm#1037-1058

Reply:
And actually it doesn't expect a `Date` object to come in in the first place. Where did you see something passing in visits array of `Date`s to do it here?